package.json

```
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "build": "remix build && tsc -b",
-    "dev": "concurrently \"remix run\" \"tsc -b -w\" \"cross-env NODE_ENV=development nodemon -w app -w loaders -e js,ts,tsx server.js\"",
+    "dev": "concurrently --names \"remix,server\" --prefix-colors \"magenta,green\" \"remix run\" \"tsc -b -w\" \"cross-env NODE_ENV=development nodemon -w app -w loaders -e js,ts,tsx server.js\"",
```

`nodemon` remains unnamed, and will show up as [2]. Add another `server` to the list of names.
```suggestion
    "dev": "concurrently --names \"remix,server,server\" --prefix-colors \"magenta,green\" \"remix run\" \"tsc -b -w\" \"cross-env NODE_ENV=development nodemon -w app -w loaders -e js,ts,tsx server.js\"",
```

---

A new release is coming soon which I think will change this starter considerably 😅